### ISSUE TITLE
camera zoom bug (MAX)

### ISSUE THREAD

**Issue by ZD-ZMA** [TheGameCreators/GameGuruRepo#5812]
camera in editor is stuck at max speed. (as if im pressing shift when im not) new keyboard doesnt fix this, holding left control doesnt slow it down either like its supposed to.

**Comment by Kasseyus** [comment]
@ZD-ZMA I have tested for this, but am so far unable to reproduce the effect.

Is this still happening for you?  

If so..

> Is it a constant issue, or is it intermittent?
> If you have access to another machine, does the issue present itself on both computers, or just this one?
> Do the Shift/L-CTRL keys function normally in other applications?

**Comment by ZD-ZMA** [comment]
> [@ZD-ZMA](https://github.com/ZD-ZMA) I have tested for this, but am so far unable to reproduce the effect.
> 
> Is this still happening for you?
> 
> If so..
> 
> > Is it a constant issue, or is it intermittent?
> > If you have access to another machine, does the issue present itself on both computers, or just this one?
> > Do the Shift/L-CTRL keys function normally in other applications?

yes, constant, only happens with game guru max. no issue with any other game and doesnt happen with classic

**Comment by LeeBamberTGC** [comment]
@ZD-ZMA Can you make a small video showing you moving the camera around an empty terrain, say the mountains biome. Note that when you are HIGH UP, the camera is designed to move very fast.

**Comment by ZD-ZMA** [comment]
> [@ZD-ZMA](https://github.com/ZD-ZMA) Can you make a small video showing you moving the camera around an empty terrain, say the mountains biome. Note that when you are HIGH UP, the camera is designed to move very fast.

im aware that high up its "supposed" to be fast. i wish that was actually the case for me though. i can certainly record a video later. busy today and tomorrow so it will probably be thursday. sorry about that.

**Comment by LeeBamberTGC** [comment]
NOTE: Just going through the older 2024 bug reports marked WAITING FOR MORE INFO.  Feel free to add new info here using the latest DEVTEST build. If nothing new is posted, I will assume it cannot be reproduced with the newer updates and close this one in the near future.

**Comment by ZD-ZMA** [comment]
although this is still an issue when zoomed out from long distance, when you get closer, the speed is more noticable... it seems slow when your far away, but it might just FEEL slow given the distance from the ground and how slow it feels to make it from one destination to another. it it possible to allow a user to change camera speed in the setting of max? maybe as advanced settings?

something like a slider/

**Comment by LeeBamberTGC** [comment]
NOTE: Marked as a feature request so we can configure a good system to control different speeds for different scenarios.
